include/boost/serialization/strong_typedef.hpp

```diff
@@ -10,6 +10,7 @@
 // strong_typedef.hpp:
 
 // (C) Copyright 2002 Robert Ramey - http://www.rrsd.com . 
+// (C) Copyright 2016 Ashish Sadanandan
```

Comment:
I'll merge it in with your copyright so that you can take the blame if 
it breaks something.

Robert Ramey

On 5/3/16 11:42 PM, Ashish Sadanandan wrote:
>
> In include/boost/serialization/strong_typedef.hpp 
> <https://github.com/boostorg/serialization/pull/40#discussion_r61994760>:
>
> > @@ -10,6 +10,7 @@
> >  // strong_typedef.hpp:
> >
> >  // (C) Copyright 2002 Robert Ramey - http://www.rrsd.com .
> > +// (C) Copyright 2016 Ashish Sadanandan
>
> I'm not sure whether I was supposed to add to the copyright or not. If 
> this isn't the right thing to do, then let me know and I'll change it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/boostorg/serialization/pull/40/files/e32c5c5e2888b7226438c67b4038a81ae31266c3#r61994760>
>

-- 
Robert Ramey
www.rrsd.com
(805)569-3793